Makefile

```diff
@@ -982,7 +982,7 @@ install.docker-full: install.docker install.docker-docs
 
 .PHONY: install.systemd
 ifneq (,$(findstring systemd,$(BUILDTAGS)))
-PODMAN_UNIT_FILES = contrib/systemd/auto-update/podman-auto-update.service \
+PODMAN_UNIT_FILES = contrib/systemd/system/podman-auto-update.service \
```

Comment:
This change has potential implications on packaging; depending on the usage of the make target.  Can you add another entry to the changelog highlighting the new home of the files just to be extra safe?

Reply:
yes sure, I think all users must use the Makefile target anyway as otherwise we only have the `service.in` template file in the git repo and you need the makefile to replace the string with the actual podman binary and generate the real unit .